{"name": "Тест неподтверждения отправки нового заказа с хэдера", "status": "failed", "statusDetails": {"message": "AssertionError: Input 'А' is invalid: Введите корректную фамилию", "trace": "self = <test_order.TestOrder object at 0x7f708f8190d0>, webdriver = <selenium.webdriver.firefox.webdriver.WebDriver (session=\"30c8b5a0-8f82-46ab-a52e-378fe2f5ecf1\")>\n\n    @allure.title('Тест неподтверждения отправки нового заказа с хэдера')\n    @allure.description('Переходим к созданию нового заказа через кнопку в хэдере, \\\n                        заполняем необходимые поля и **не** подтверждаем отправку заказа, \\\n                        проверяем, что вернулись на форму заполнения информации об аренде')\n    def test_order_from_header_not_confirm(self, webdriver: WebDriver):\n        base_page = BasePage(webdriver)\n    \n        main_page = MainPage(webdriver, base_page)\n        main_page.click_button_order()\n    \n        order_customer_page = OrderCustomerPage(webdriver, base_page)\n>       order_customer_page.fill_form()\n\ntests/test_order.py:72: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/order_customer_page.py:35: in fill_form\n    self.check_valid_last_name()\npages/order_customer_page.py:88: in check_valid_last_name\n    self.__check_valid_input(self.input_last_name)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <pages.order_customer_page.OrderCustomerPage object at 0x7f708f6851d0>, locator = ['xpath', './/input[starts-with(@class, \"Input_Input\") and contains(@placeholder, \"Фамилия\")]']\n\n    def __check_valid_input(self, locator: List[str]):\n        input_element = self.webdriver.find_element(*locator)\n    \n        given_value = input_element.get_attribute('value')\n    \n        input_element.clear()\n        input_element.send_keys(given_value)\n    \n        error_locator = [By.XPATH, f'{locator[1]}/../div']\n    \n        error_element = self.webdriver.find_element(*error_locator)\n    \n        is_valid = len(error_element.text) == 0\n    \n>       assert is_valid, f'Input \\'{given_value}\\' is invalid: {error_element.text}'\nE       AssertionError: Input 'А' is invalid: Введите корректную фамилию\n\npages/order_customer_page.py:117: AssertionError"}, "description": "Переходим к созданию нового заказа через кнопку в хэдере,                         заполняем необходимые поля и **не** подтверждаем отправку заказа,                         проверяем, что вернулись на форму заполнения информации об аренде", "steps": [{"name": "Клик на кнопку Заказать", "status": "passed", "start": 1681979884506, "stop": 1681979884742}, {"name": "Заполнение информации о заказчике", "status": "failed", "statusDetails": {"message": "AssertionError: Input 'А' is invalid: Введите корректную фамилию\n", "trace": "  File \"/home/user/yandex-practicum/Sprint_4/.venv/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/user/yandex-practicum/Sprint_4/pages/order_customer_page.py\", line 35, in fill_form\n    self.check_valid_last_name()\n  File \"/home/user/yandex-practicum/Sprint_4/.venv/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/user/yandex-practicum/Sprint_4/pages/order_customer_page.py\", line 88, in check_valid_last_name\n    self.__check_valid_input(self.input_last_name)\n  File \"/home/user/yandex-practicum/Sprint_4/pages/order_customer_page.py\", line 117, in __check_valid_input\n    assert is_valid, f'Input \\'{given_value}\\' is invalid: {error_element.text}'\n"}, "steps": [{"name": "Ввод информации о заказчике: имя", "status": "passed", "parameters": [{"name": "value", "value": "'Устинович'"}], "start": 1681979884743, "stop": 1681979884905}, {"name": "Проверка валидности ввода: имя", "status": "passed", "start": 1681979884905, "stop": 1681979884988}, {"name": "Ввод информации о заказчике: фамилия", "status": "passed", "parameters": [{"name": "value", "value": "'Аникей'"}], "start": 1681979884988, "stop": 1681979885003}, {"name": "Проверка валидности ввода: фамилия", "status": "failed", "statusDetails": {"message": "AssertionError: Input 'А' is invalid: Введите корректную фамилию\n", "trace": "  File \"/home/user/yandex-practicum/Sprint_4/.venv/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/user/yandex-practicum/Sprint_4/pages/order_customer_page.py\", line 88, in check_valid_last_name\n    self.__check_valid_input(self.input_last_name)\n  File \"/home/user/yandex-practicum/Sprint_4/pages/order_customer_page.py\", line 117, in __check_valid_input\n    assert is_valid, f'Input \\'{given_value}\\' is invalid: {error_element.text}'\n"}, "start": 1681979885003, "stop": 1681979885050}], "start": 1681979884742, "stop": 1681979885051}], "attachments": [{"name": "test_order_from_header_not_confirm.png", "source": "1ccb74b5-a510-44bc-b6f0-32ab98d9272a-attachment.png", "type": "image/png"}], "start": 1681979884506, "stop": 1681979885052, "uuid": "d0ff7e40-1239-4dc1-903c-fbff32544ea6", "historyId": "8a77859421967105cb067b0359421618", "testCaseId": "87e587908bf5abd3b47ef7b9be612e24", "fullName": "tests.test_order.TestOrder#test_order_from_header_not_confirm", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_order"}, {"name": "subSuite", "value": "TestOrder"}, {"name": "host", "value": "workstation"}, {"name": "thread", "value": "91721-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_order"}]}